### PR TITLE
[SDPA-856] card carousel

### PR DESCRIPTION
**Pull request by alan-cole**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-856

### Changed

1.  Added Card Carousel to ripple Card.

### Screenshots

screen shot 2018-09-13 at 6 51 27 pm
screen shot 2018-09-13 at 6 51 19 pm
screen shot 2018-09-13 at 6 51 03 pm

### CODE REVIEW

File: src/storybook-components/Page.vue [hunk 1]
```diff
@@ -99,6 +99,9 @@
       </template>
 
       <template slot="belowContent">
+        <div class="rpl-site-constrain--on-all">
+          <rpl-card-carousel :title="mock.cardCarousel.title" :cards="mock.cardCarousel.cards" />
```
Comment (tim-yao) [inline src/storybook-components/Page.vue]: Hi @alan-cole can you please place this component in content slot? it suppose to be a landing page body component.

File: packages/Molecules/Card/CardCarousel.vue [hunk 2]
```diff
@@ -0,0 +1,210 @@
+<template>
+  <div class="rpl-carousel">
```
Comment (tim-yao) [inline packages/Molecules/Card/CardCarousel.vue]: Could you please correct the class name to match the component name? It should be `rpl-card-carousel`.

File: packages/Molecules/Card/CardCarousel.vue [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+import RplIcon from '@dpc-sdp/ripple-icon'
+import { Carousel, Slide } from 'vue-carousel'
+
+import RplCardPromotion from './CardPromotion.vue'
+import RplCardEvent from './CardEvent.vue'
+import RplCardKeydates from './CardKeydates.vue'
+// TODO: Add future card components
+
+export default {
+  name: 'RplCarousel',
```
Comment (tim-yao) [inline packages/Molecules/Card/CardCarousel.vue]: Same, the name need to be `RplCardCarousel`.

Comment (alan-cole) [reply packages/Molecules/Card/CardCarousel.vue]: Fixed.

File: packages/Molecules/Card/CardCarousel.vue (hunk 2 above)
Comment (alan-cole) [reply packages/Molecules/Card/CardCarousel.vue]: Fixed.

File: src/storybook-components/Page.vue (hunk 1 above)
Comment (alan-cole) [reply src/storybook-components/Page.vue]: Updated - please note that it is now affected by the sidebar. Sidebar should be disabled to correctly view.

**Pull request merged**